app/views/shared/dialogs/_dialog_field_drop_down_list.html.haml

```diff
@@ -1,7 +1,13 @@
 - if edit
-  - select_values = field.values.collect(&:reverse)
+  - select_values = field.values.collect
```

Comment:
the (&:reverse) seems to be important, without that, the names and ids in the selectboxes have switched